renter/renterutil/hostset.go

```diff
@@ -96,15 +98,18 @@ func (set *HostSet) HasHost(hostKey hostdb.HostPublicKey) bool {
 
 // Close closes all of the sessions in the set.
 func (set *HostSet) Close() error {
+	var err error
 	for hostKey, lh := range set.sessions {
 		lh.mu.Lock()
 		if lh.s != nil {
-			lh.s.Close()
+			if e := lh.s.Close(); e != nil {
+				err = multierror.Append(err, e)
+			}
```

Comment:
These errors aren't really actionable either, but I guess we should leave that up to the caller of `(HostSet).Close`. We can use `HostErrorSet` for this.